### PR TITLE
[liferay-paypal-workspace] switch to static CX 

### PR DESCRIPTION
**Pull request by gamerson**

### CODE REVIEW

File: workspaces/liferay-paypal-workspace/client-extensions/liferay-paypal-commerce-checkout-step/LCP.json [hunk 1]
```diff
@@ -13,18 +19,19 @@
...
 	},
-	"memory": 512,
+	"memory": 50,
 	"readinessProbe": {
 		"httpGet": {
-			"path": "/ready",
-			"port": 58081
+			"path": "/",
+			"port": 80
 		}
```
Comment (brianikim) [inline workspaces/liferay-paypal-workspace/client-extensions/liferay-paypal-commerce-checkout-step/LCP.json:35]: @gamerson ,

If we remove the Spring Controllers, how would we support the following ?

https://github.com/liferay/liferay-portal/blob/c5ede076937e208feae3b423e5e5b54883a68bce/modules/apps/commerce/commerce-client-extension-web/src/main/java/com/liferay/commerce/client/extension/web/internal/util/ClientExtensionCommerceCheckoutStep.java#L115

https://github.com/liferay/liferay-portal/blob/c5ede076937e208feae3b423e5e5b54883a68bce/modules/apps/commerce/commerce-client-extension-web/src/main/java/com/liferay/commerce/client/extension/web/internal/util/ClientExtensionCommerceCheckoutStep.java#L183

Both the sample checkout step and my checkout step are reliant on the Liferay/Server side as part of ClientExtensionCommerceCheckoutStep.

My code is not reliant on an ActionRestController and/or /action being available, but other CX checkout steps might be as seen in our sample checkout step here: https://github.com/liferay/liferay-portal/blob/c5ede076937e208feae3b423e5e5b54883a68bce/workspaces/liferay-sample-workspace/client-extensions/liferay-sample-commerce-checkout-step/src/main/java/com/liferay/sample/ActionRestController.java#L35

**Comment by brianikim** [comment]
@kevenleone ,

You are also going to need my second commit from here:
https://github.com/brianikim/liferay-portal/tree/commerceStepStaticAdditional


Commit: https://github.com/liferay/liferay-portal/commit/a8b6942441832fa610cd6374f800cb5d2e068df7

This needs to be deployed in order for our test to properly continue.

File: workspaces/liferay-paypal-workspace/client-extensions/liferay-paypal-commerce-checkout-step/LCP.json (hunk 1 above)
Comment (brianikim) [reply workspaces/liferay-paypal-workspace/client-extensions/liferay-paypal-commerce-checkout-step/LCP.json:35]: I tried testing the `catapult.launch` check with the revised ports, but it fails to work even after I change the path to `/ready` etc.

**Comment by kevenleone** [comment]
@brianikim as our environment used is a Liferay SaaS I cannot deploy the change you made

In order to test I need this to be merged and then after a new release is made I can upgrade the DXP version, currently is 2024@Q3

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Pull request closed without merging**